cliutil/format.go

```diff
@@ -2,24 +2,28 @@ package cliutil
 
 import (
 	"fmt"
+	"strings"
 
 	"github.com/Masterminds/semver"
 )
 
-func Format(nextVersion semver.Version, hasNextVersion bool, format string) []string {
+func Format(nextVersion semver.Version, hasNextVersion bool, format string, prefix ...string) []string {
+	prefixString := strings.Join(prefix, "")
```

Comment:
This also means we can get rid of this line. However, this means that we need to change where the function gets called, so that (by default) an empty string is handed over. Anyway, this should not be hard-coded, but an empty string should just be the default value of the CLI flag.